electron/src/main.ts

```diff
@@ -652,7 +652,7 @@ class ElectronWrapperInit {
             params.plugins = 'false';
             webPreferences.allowRunningInsecureContent = false;
             webPreferences.contextIsolation = false;
-            webPreferences.experimentalFeatures = true;
+            webPreferences.experimentalFeatures = false;
```

Comment:
why setting experimental features to false also?

Reply:
because we don't need them, and it is a safer default.

https://www.electronjs.org/docs/latest/tutorial/security#9-do-not-enable-experimental-features